PWGCF/FemtoDream/FemtoDreamTrackSelection.h

```diff
@@ -268,29 +274,29 @@ bool FemtoDreamTrackSelection::isSelectedMinimal(T const& track)
   }
 
   /// check whether the most open cuts are fulfilled
-  const static int nPtMinSel = getNSelections(femtoDreamTrackSelection::kpTMin);
-  const static int nPtMaxSel = getNSelections(femtoDreamTrackSelection::kpTMax);
-  const static int nEtaSel = getNSelections(femtoDreamTrackSelection::kEtaMax);
-  const static int nTPCnMinSel = getNSelections(femtoDreamTrackSelection::kTPCnClsMin);
-  const static int nTPCfMinSel = getNSelections(femtoDreamTrackSelection::kTPCfClsMin);
-  const static int nTPCcMinSel = getNSelections(femtoDreamTrackSelection::kTPCcRowsMin);
-  const static int nTPCsMaxSel = getNSelections(femtoDreamTrackSelection::kTPCsClsMax);
-  const static int nDCAxyMaxSel = getNSelections(femtoDreamTrackSelection::kDCAxyMax);
-  const static int nDCAzMaxSel = getNSelections(femtoDreamTrackSelection::kDCAzMax);
-  const static int nDCAMinSel = getNSelections(femtoDreamTrackSelection::kDCAMin);
-  const static int nPIDnSigmaSel = getNSelections(femtoDreamTrackSelection::kPIDnSigmaMax);
-
-  const static float pTMin = getMinimalSelection(femtoDreamTrackSelection::kpTMin, femtoDreamSelection::kLowerLimit);
-  const static float pTMax = getMinimalSelection(femtoDreamTrackSelection::kpTMax, femtoDreamSelection::kUpperLimit);
-  const static float etaMax = getMinimalSelection(femtoDreamTrackSelection::kEtaMax, femtoDreamSelection::kAbsUpperLimit);
-  const static float nClsMin = getMinimalSelection(femtoDreamTrackSelection::kTPCnClsMin, femtoDreamSelection::kLowerLimit);
-  const static float fClsMin = getMinimalSelection(femtoDreamTrackSelection::kTPCfClsMin, femtoDreamSelection::kLowerLimit);
-  const static float cTPCMin = getMinimalSelection(femtoDreamTrackSelection::kTPCcRowsMin, femtoDreamSelection::kLowerLimit);
-  const static float sTPCMax = getMinimalSelection(femtoDreamTrackSelection::kTPCsClsMax, femtoDreamSelection::kUpperLimit);
-  const static float dcaXYMax = getMinimalSelection(femtoDreamTrackSelection::kDCAxyMax, femtoDreamSelection::kAbsUpperLimit);
-  const static float dcaZMax = getMinimalSelection(femtoDreamTrackSelection::kDCAzMax, femtoDreamSelection::kAbsUpperLimit);
-  const static float dcaMin = getMinimalSelection(femtoDreamTrackSelection::kDCAMin, femtoDreamSelection::kAbsLowerLimit);
-  const static float nSigmaPIDMax = getMinimalSelection(femtoDreamTrackSelection::kPIDnSigmaMax, femtoDreamSelection::kAbsUpperLimit);
+  int nPtMinSel = getNSelections(femtoDreamTrackSelection::kpTMin);
```

Comment:
You removed the static. This means this will be executed once per track and therefore will be much slower. Why the static is gone?

Reply:
The function isSelectedMinimal for Track objects is also called in the V0selection class for the daughters.
The number of selections (e.g nPtMinSel) and the minimal values (e.g pTMin) for tracks and V0 daughters are different. 
And with the const static types they were kept set to the Track values, making impossible to obtain candidates V0s.
So I removed in that function all the const static. I first removed const, keeping static but the problem was still there.

Reply:
I understand but this will be extremely slow. 

You have to find a different solution then. E.g. a class which is then instantiated twice. Once for tracks and once for V0s. It sets its values once at the beginning, but not for each track/V0

Reply:
@jgrosseo ok working on it.
Thanks